package.json

```diff
@@ -20,7 +20,7 @@
 		"appium": "^1.20.2",
 		"babel-core": "^7.0.0-bridge.0",
 		"babel-eslint": "^8.2.2",
-		"babel-jest": "^24.9.0",
+		"babel-jest": "^27.4.5",
```

Comment:
This version matches the value from Gutenberg's `package.json` file:
https://github.com/WordPress/gutenberg/blob/5a84b27bcead1528ca98bcade9c6d5259c70de7b/package.json#L151